src/ui/components/FormOverlay/index.js

```diff
@@ -152,10 +152,7 @@ export class FormOverlayBase extends React.Component<Props> {
   }
 }
 
-const mapStateToProps = (
-  state: {| formOverlay: FormOverlayState |},
-  ownProps: Props,
-): $Shape<Props> => {
+const mapStateToProps = (state: AppState, ownProps: Props): $Shape<Props> => {
```

Comment:
Ah, I see here's another example of specifying a return type, but using `$Shape<Props>` rather than an explicit type.

Reply:
That was there before too 🤷‍♂️

Reply:
Right, well if we don't need these, and this is a cleanup patch, we might as well clean them up.

Reply:
I do agree.